### PR TITLE
Added lock/unlock feature for quorum

### PR DESCRIPTION
**Pull request by himanshulalarya**
Signed-off-by: Lal <himanshu.a.lal@accenture.com>

**Changelog**
- Added lock/unlock feature for quorum nodes

 

**Reviewed by**
@sownak

 

**Linked issue**
#543 

Commit message
-------------
Added lock/unlock feature for quorum

### CODE REVIEW

File: platforms/quorum/charts/node_constellation/templates/deployment.yaml [hunk 1]
```diff
@@ -204,24 +204,45 @@ spec:
...
-           --rpc \
-           --rpcaddr 0.0.0.0 \
-           --rpcport {{ .Values.node.ports.rpc }} \
-           --rpcvhosts=* \
-           --port {{ .Values.node.ports.quorum }} \
-           --unlock 0 \
-           --password $QUORUM_DATA_DIR/password.txt 2>&1 | tee -a $QUORUM_HOME/logs/quorum.log;
+          
+          LOCK_STATUS={{ .Values.node.lock }}
+          if [ $LOCK_STATUS = true ]
```
Comment (sownak) [inline platforms/quorum/charts/node_constellation/templates/deployment.yaml:209]: use code similar to $args, no need to put all the code in if-else

File: platforms/quorum/charts/node_tessera/templates/deployment.yaml [hunk 2]
```diff
@@ -297,26 +297,47 @@ spec:
...
-          --emitcheckpoints \
-          --rpc \
-          --rpcaddr 0.0.0.0 \
-          --rpcport {{ .Values.node.ports.rpc }} \
-          --rpcvhosts=* \
-          --port {{ .Values.node.ports.quorum }} \
-          --unlock 0 \
-          --password $QUORUM_DATA_DIR/password.txt 2>&1 | tee -a $QUORUM_HOME/logs/quorum.log; 
+          LOCK_STATUS={{ .Values.node.lock }}
+          if [ $LOCK_STATUS = true ]
```
Comment (sownak) [inline platforms/quorum/charts/node_tessera/templates/deployment.yaml:304]: same comment as previous

**Comment by sownak** [comment]
> Also create a PR for baf-configuration so that builds don't fail.

@himanshulalarya This comment has not been addressed yet.

**Comment by himanshulalarya** [comment]
> > Also create a PR for baf-configuration so that builds don't fail.
> 
> @himanshulalarya This comment has not been addressed yet.

Done.

**Pull request merged**